src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -12,17 +13,10 @@
     "protobuf": ("google.protobuf",),
     "pycrypto": ("Crypto",),
     "pyopenssl": ("OpenSSL",),
-    "python-dateutil": ("dateutil",),
-    "python-jose": ("jose",),
-    "PyYAML": ("yaml",),
-    "pymongo": (
-        "bson",
-        "gridfs",
-    ),
-    "pytest_runner": ("ptr",),
     "python_dateutil": ("dateutil",),
-    "setuptools": (
-        "easy_install",
-        "pkg_resources",
-    ),
+    "python_jose": ("jose",),
```

Comment:
Its probably fine to normalize these by hand, but it would be more robust to be passing these all through the normalization function here or else at the point of use.